server/game/cards/attachments/07/catapultonthewall.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class CatapultOnTheWall extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Kneel Catapult and attached character',
+            condition: () => this.game.currentChallenge,
+
+            // This is not a nice interaction for users, but `ability.costs` would need to be updated with a different method to allow something like attachment.parent.kneel() (I think)
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.kneel(card => card === this.parent)
```

Comment:
I apologize for not being clearer, you should also switch this to `ability.costs.kneelParent()` now that you've pulled in the cost implementation.